dockerfiles/node-bench-regression-guard/node-bench-regression-guard

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+
```

Comment:
is it OK that the file has no ext?

Reply:
Basically yes, the file is already marked as the executable and has its own shebang line.